felix/dataplane/linux/endpoint_mgr.go

```diff
@@ -975,6 +1080,11 @@ func (m *endpointManager) updateHostEndpoints() {
 		}
 	}
 
+	ifaceNameToPolicyGroups := map[string][]*rules.PolicyGroup{}
```

Comment:
Ah yes, I just copied the old inelegant solution for policies.  Could do a bit better here because we do know which groups go in which tables; just seemed like the number of tracking maps would explode a bit

Reply:
I would personally be fine with leaving this to a future tidy up, as this PR is already a good unit on its own.